### DISCUSSION TITLE
Windows PE 3.0 works (bascially just windows 7)

### DISCUSSION THREAD

**Discussion by Alsefina** [copy/v86#951] [Show and tell]
After reading #892 and seeing that Windows 8 works on v86, I was pretty impressed. Windows 8 ran pretty bad so I thought to myself "Gee, if only Microsoft made a stripped-down barebones version of Windows 8 that ran straight from a CD". So then I tried every single version of Windows PE starting from the latest. Eventually, PE 3.0 seemed to be the one that ran perfectly without issues.

VM config I used: 
ram: 2048MB
vram: 256MB
ACPI enabled

After booting, it should show the iconic windows 7 wallpaper and then start installing various drivers of all kinds (give it a moment and a cmd window should appear)
<img width="1323" alt="Screenshot 2023-12-03 at 3 00 28 PM" src="https://github.com/copy/v86/assets/93541947/f37d41a7-941d-4f08-8ef7-98af2fc26877">

desktop
<img width="1323" alt="Screenshot 2023-12-03 at 3 08 25 PM" src="https://github.com/copy/v86/assets/93541947/350e1e05-f142-4c2e-adfb-05ec51c44baf">

ill figure out networking later 
<img width="1323" alt="Screenshot 2023-12-03 at 3 12 58 PM" src="https://github.com/copy/v86/assets/93541947/7c017b51-c801-4af3-bb55-5368aa727b50">

I got the iso image here: https://archive.org/details/windows-7-pe 
haven't tried the SWT_PE iso that the uploader made, but it should work

it would be pretty cool if someone managed to get the windows 10/11 versions of PE working.

**Comment by CorruptionVR** [comment]
ok. so i tried the SWT_pe.iso file, and I get this error
![download](https://github.com/copy/v86/assets/78228404/dc1b7ebf-3e99-4412-811e-26c32f97ba36)

**Reply by CorruptionVR** [reply]
ok, so, I went to Windows 2000, and saw that it used the ipconfig command, so I tried `ipconfig /release *` and `ipconfig /renew *`, but I got this error
![download](https://github.com/copy/v86/assets/78228404/2b75b375-cf45-4172-a023-45bd9de84920)

**Reply by copy** [reply]
Windows 7 doesn't support ne2k, see https://github.com/copy/v86/issues/473

**Reply by SuperMaxusa** [reply]
> Windows 7 doesn't support ne2k, see #473

Officially - no, but it's can supported with driver for Windows 2000: https://github.com/copy/v86/issues/392#issuecomment-1402659862
Not sure how this driver install on Windows PE, probably also via devmgmt or something else.

**Reply by 36479756** [reply]
you can try using a lite verson on qemu then install driver for windows 2000 i guess
